core/src/main/java/com/graphhopper/util/EdgeIteratorState.java

```diff
@@ -26,6 +26,8 @@
  */
 public interface EdgeIteratorState
 {
+    final static int DISPREFERED_STARTSTOPEDGE = -1;
```

Comment:
I would not put it into QueryGraph, but Weighting seems also a good location for me